### DISCUSSION TITLE
Use DVC for more than just ML models?

### DISCUSSION THREAD

**Discussion by evbo** [iterative/dvc.org#3527] [Ideas]
Has DVC considered expanding scope to include other industries that need to version control big datasets? For instance, [EDA](https://en.wikipedia.org/wiki/Electronic_design_automation) is one example that has enormous schematic files and terabytes of simulations, all needing to be carefully paired with various config and scripts. DVC sounds like it could serve many useful purposes! Or is it intentionally wanting to remain focused on just ML?

**Comment by rogermparent** [comment]
We have actually considered DVC as a possible solution to manage image files in our website engine! See #1115 for some discussion on that end if you're interested. While SSG image management is a very different use case from what you describe here, us considering DVC for this purpose does indicate that we're not staunchly set on DVC being an ML-only tool forever even if that's the primary use case.

**Comment by jorgeorpinel** [comment]
Given [our company](https://iterative.ai/)'s mission, I think our docs should stay focused on ML Engineering use cases but yes, definitely other industries beyond Data Science can benefit from DVC features, especially [data management](https://dvc.org/doc/start/data-management).

We do have a [page about related technologies](https://dvc.org/doc/user-guide/related-technologies) which could be expanded to cover other industries just to have some official opinion on this matter. Other than that, I'm not sure this is a docs discussion really. Should we transfer it as a question or discussion in https://github.com/iterative/dvc ?

**Reply by evbo** [reply]
Sounds good to me!

**Reply by daavoo** [reply]
There is a `beyond-ml` channel in the community discord (https://discordapp.com/invite/dvwXA2N) to discuss these kinds of topics

**Reply by evbo** [reply]
Thanks! I posted a starter topic in there to hopefully pique the interests of others: https://discord.com/channels/485586884165107732/918159153824952320/974323452347572264
